instrumentation/packaging/fpm/splunk-otel-javaagent.properties

```
@@ -0,0 +1,17 @@
+# System properties for the Splunk OpenTelemetry Java Instrumentation Agent
+
+# *NOTE*: The system properties defined in this file may be overriden by the corresponding environment variables if
+#   configured at the system level, the /etc/systemd/system.conf.d/splunk-otel-javaagent.conf drop-in file,
+#   or by an individual service.
+
+# To add/modify the default system properties for the Java instrumentation agent (requires root privileges):
+#  1. See https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.html
+#     for details about supported system properties.
+#  2. Add/Modify the desired system properties below and save this file.
+#  3. Restart all applicable services with "sudo systemctl restart <service-name>".
+
+otel.service.name=
+otel.resource.attributes=
+splunk.metrics.enabled=false
+splunk.profiler.enabled=false
+splunk.profiler.memory.enabled=false
```

I would prefer to leave these out so that the agent defaults come into play. Otherwise, we will have two different places where defaults are maintained (here and the agent), and keeping those two in sync could prove burdensome and potentially confusing to users.

```suggestion
#otel.service.name=
#otel.resource.attributes=
#splunk.metrics.enabled=false
#splunk.profiler.enabled=false
#splunk.profiler.memory.enabled=false
```

---

Good point, thanks!